client/layout/masterbar/logged-out.jsx

```diff
@@ -129,9 +121,11 @@ class MasterbarLoggedOut extends React.Component {
 				 */
 				signupUrl = currentQuery.redirect_to;
 			} else {
+				// TODO: should it be changed to '/start' or '/jetpack/connect'?
 				signupUrl = '/jetpack/new';
 			}
 		} else if ( 'jetpack-connect' === sectionName ) {
+			// TODO: should it be changed to '/start' or '/jetpack/connect'?
```

Comment:
Same as https://github.com/Automattic/wp-calypso/pull/45580/files#r487694898

Reply:
Changed to `/jetpack/connect`